### PR TITLE
ci: Switch to the MSVC toolchain on Windows and get rid of msys2 comp…

### PR DESCRIPTION
**Pull request by sdroege**
…lications

----

Another try at https://github.com/rust-av/dav1d-rs/pull/76 but it seems like the Windows CI images are quite broken :scream:

### CODE REVIEW

File: .github/workflows/dav1d.yml [hunk 1]
```diff
@@ -49,32 +49,58 @@ jobs:
...
+        pip install -U pip
+        pip install -U wheel setuptools
+        pip install -U meson ninja
+
+    - name: Setting up environment
+      run: |
+        echo "PKG_CONFIG=c:\build\bin\pkg-config.exe" >> $GITHUB_ENV
+        echo "PKG_CONFIG_PATH=C:\build\lib\pkgconfig" >> $GITHUB_ENV
+        echo "C:\build\bin" >> $GITHUB_PATH
+      shell: bash
```
Comment (Luni-4) [inline .github/workflows/dav1d.yml:86]: Can we move `shell: bash` after `name` field?

**Review by Luni-4 — CHANGES_REQUESTED** [review]
Thank you @sdroege! Just a simple change to immediately see we are not using `PowerShell`

File: .github/workflows/dav1d.yml (hunk 1 above)
Comment (sdroege) [reply .github/workflows/dav1d.yml]: Sure, not sure if that works though. Let's try. All examples I saw had it in the end.

Comment (sdroege) [reply .github/workflows/dav1d.yml]: Works apparently too

**Review by Luni-4 — APPROVED** [review]

**Pull request merged**